src/components/Tasks/Tasks.tsx

```diff
@@ -70,19 +69,40 @@ const Tasks = () => {
       ? filterTasks(isUrgent)(selectedCategory)
       : tasks;
 
-  const displayTasks = filteredTasks.map((item) => {
-    return (
-      <Task
-        key={item.task}
-        {...item}
-        markCompleted={markCompleted}
-        deleteTask={deleteTask}
-      />
-    );
-  });
+   //useMemo
+  const displayTasks = useMemo(()=>{
```

Comment:
This is something that I have missed in the last PR. 
We use verbs for functions.
Since this isn't a function, it should be named differently.
One way would be:
```suggestion
  const buddies = useMemo(()=>{
```

Reply:
Oh okay. I asked you about that naming convetion in WorkBuddies.tsx , but now I get it